### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Screen Reader: NVDA/JAWS is reading the column header name and column values wrongly

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/vscode-mssql#1226]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11ymssqlforVSCode; #MAS1.3.1; #C+E-H; #mssqlforVSCode; #ScreenReader; #NVDA; #JAWS

**Environment Details:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0)
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.18362 Build 18362)
Screen reader: NVDA, JAWS

**MAS Violated:** MAS 1.3.1

**Repro Steps:** 
1.Open Visual Studio   Code in your system
2.Navigate to   "Extensions" and install "SQl Server (mssql)" extension
3.Press F1 and   select MS SqL : Connect and connect to any sql server
4.Write code (SELECT   * FROM sys.objects;) in the new document and press F1 and select   ""MS SQL : Execute Query"" "  
5.Start NVDA/JAWS   
6.Navigate to   results table grid with the help of mouse   
7.Verify if   NVDA/JAWS is reading the table elements correctly

**Actual Result:** 
NVDA/JAWS is reading the column header name and column values wrongly 

**Expected Result:** 
Screen reader should read the related column names and column number properly when focus is on the table elements

**User Impact:** 
Screen Reader users will be provided with faulty data if the screen readers will read column header name and column values wrongly

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_1226_NVDA not reading table properly.pptx](https://github.com/Microsoft/vscode-mssql/files/3110941/Bug_1226_NVDA.not.reading.table.properly.pptx)

**Comment by kenvanhyning** [comment]
This should be fixed in 1.7, please verify and reopen if any issues are found.

**Comment by Kothoor** [comment]
BM_MSSQL_MAY2020

**Comment by fsteffi** [comment]
#Closed;

repro'ing still hence logged a new bug https://github.com/microsoft/vscode-mssql/issues/17712
